src/vs/workbench/contrib/positronAssistant/browser/positronAssistantService.ts

```diff
@@ -77,10 +82,19 @@ export class PositronAssistantService extends Disposable implements IPositronAss
 
 	showLanguageModelModalDialog(
 		sources: IPositronLanguageModelSource[],
-		onSave: (config: IPositronLanguageModelConfig) => Promise<void>,
+		onAction: (config: IPositronLanguageModelConfig, action: string) => Promise<void>,
 		onCancel: () => void,
+		onClose: () => void,
 	): void {
-		showLanguageModelModalDialog(this._keybindingService, this._layoutService, sources, onSave, onCancel);
+		showLanguageModelModalDialog(this._keybindingService, this._layoutService, this._configurationService, sources, onAction, onCancel, onClose);
+	}
+
+	getSupportedProviders(): string[] {
+		const providers = ['anthropic', 'google', 'copilot'];
+		if (IsDevelopmentContext.getValue(this._contextKeyService)) {
```

Comment:
Could you add a config value that would cause this to happen in release builds too? Then you could close #6876 with this change, too.

Reply:
Will do!